client/app/services/intercom.js

```diff
@@ -23,7 +29,16 @@ export default Ember.Service.extend({
     /* eslint-enable camelcase */
   },
 
+  trackEvent(eventName) {
+    let eventAttribute = Ember.String.camelize(eventName);
+    if (!this.get(eventAttribute))  {
```

Comment:
I was initially confused as to what this is doing, it seems like it's a mechanism to prevent the same event from being tracked multiple times per session. Is there any downside to just letting this happen? (or handling it on a per-event basis)